matRad_DijSampling.m

```diff
@@ -35,25 +34,63 @@
 %
 % %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-if ~exist('SamplingRate','var')
-    SamplingRate       = 0.1;
+boolFineClustering  = false;                                     % flag to switch between fine and coarse radiological depth resolution
+deltaRadDepth       = 5;                                         % step size of radiological depth
+LatCutOffsq         = r0^2;
+
+%% remember dose values inside the inner core
+ixCore              = rad_distancesSq < LatCutOffsq;             % get voxels indices having a smaller radial distance than r0
+bixelDoseCore       = bixelDose(ixCore);                         % save dose values that are not affected by sampling
+
+ixTail              = ~ixCore;                                   % get voxels indices beyond r0
+linIxSample         = find(ixTail);                              % convert logical index to linear index
```

Comment:
rename to 'linIxTail' - everything has the suffix Tail. I argue renaming would be more consistent.